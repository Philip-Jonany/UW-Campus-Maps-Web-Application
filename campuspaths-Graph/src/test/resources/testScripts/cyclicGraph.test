CreateGraph A

AddNode A n1
AddNode A n2
AddNode A n3

AddEdge A n1 n2 e1
AddEdge A n2 n3 e1
AddEdge A n3 n1 e1
AddEdge A n1 n3 e2
AddEdge A n3 n2 e2
AddEdge A n2 n1 e2

ListChildren A n1