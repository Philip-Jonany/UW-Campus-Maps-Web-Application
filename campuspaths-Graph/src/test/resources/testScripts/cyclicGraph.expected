created graph A

added node n1 to A
added node n2 to A
added node n3 to A

added edge e1 from n1 to n2 in A
added edge e1 from n2 to n3 in A
added edge e1 from n3 to n1 in A
added edge e2 from n1 to n3 in A
added edge e2 from n3 to n2 in A
added edge e2 from n2 to n1 in A

the children of n1 in A are: n2(e1) n3(e2)